File: 2169.cpp
struct Solution {
    int countOperations(int num1, int num2) {
        int count = 0;
        if (num1 < num2){
            swap(num1, num2);
        }
        while (num2 != 0){
            count += num1 / num2;
            num1 %= num2;
            swap(num1, num2);
        }
        return count;
    }
};

File: 2169.py
class Solution:
    def countOperations(self, num1: int, num2: int) -> int:
        count = 0
        if num1 < num2:
            num1, num2 = num2, num1
        while num2 != 0:
            count += num1 // num2
            num1 %= num2
            num1, num2 = num2, num1
        return count
